src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -81,6 +82,11 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
                 this.meter.CreateObservableGauge($"{metricPrefix}memory.usage", () => (double)(Environment.WorkingSet / 1_000_000), "MB", "Working Set");
             }
 
+            if (options.IsProcessEnabled)
+            {
+                this.meter.CreateObservableCounter("process.cpu.time", () => Process.GetCurrentProcess().TotalProcessorTime.TotalSeconds, "s", "Total processor time of this process");
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process

mention this link in the comment for future readers reference.

Reply:
wondering if we can do UserProcessorTime and PrivilegedProcessorTime with state tag?